webapp/src/main/webapp/WEB-INF/portlet.xml

```diff
@@ -1088,4 +1088,25 @@
     </portlet-info>
   </portlet>
 
+  <portlet>
+    <portlet-name>AdminsPortlet</portlet-name>
+    <display-name>Admins Portlet</display-name>
+    <portlet-class>org.exoplatform.commons.api.portlet.GenericDispatchedViewPortlet</portlet-class>
+    <init-param>
+      <name>portlet-view-dispatched-file-path</name>
+      <value>/html/admins.html</value>
+    </init-param>
+    <expiration-cache>-1</expiration-cache>
+    <cache-scope>PUBLIC</cache-scope>
+    <supports>
+      <mime-type>text/html</mime-type>
+    </supports>
+    <supported-locale>en</supported-locale>
+    <resource-bundle>locale.portlet.Admins</resource-bundle>
```

Comment:
Please rename I18N file as well to `PlatformAdminsWidget` + Ensure to use the real location of the file since you defined it in `locale.portlet.social` rather than `locale.portlet`

```suggestion
    <resource-bundle>locale.portlet.PlatformAdminsWidget</resource-bundle>
```